cdk/PgStacInfra.ts

```diff
@@ -54,6 +62,13 @@ export class PgStacInfra extends Stack {
       ],
     });
 
+    // Pgstac Database
+    // set instance type to t3.micro if stage is test, otherwise t3.small
+    const dbInstanceType =
```

Comment:
Should we move this to a config file/parameter?

Reply:
Yeah I think that's a good idea